### PR TITLE
[JAVA][jaxrs-spec] Added custom methods to add/remove elements from java collections

### PR DESCRIPTION
**Pull request by wolfdale**
Changes are made to `pojo.mustache` of Jaxrs-spec to generate custom methods to remove or add elements from Java collection such as List, Maps.

This resolves issue #6467 

### PR checklist

- [ ] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [ ] If contributing template-only or documentation-only changes which will change sample output, [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) before.
- [ ] Run the shell script(s) under `./bin/` (or Windows batch scripts under`.\bin\windows`) to update Petstore samples related to your fix. This is important, as CI jobs will verify _all_ generator outputs of your HEAD commit, and these must match the expectations made by your contribution. You only need to run `./bin/{LANG}-petstore.sh`, `./bin/openapi3/{LANG}-petstore.sh` if updating the code or mustache templates for a language (`{LANG}`) (e.g. php, ruby, python, etc).
- [ ] File the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master`, `4.3.x`, `5.0.x`. Default: `master`.
- [ ] Copy the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) to review the pull request if your PR is targeting a particular programming language.

Technical Committee
@bbdouglas (2017/07) @sreeshas (2017/08) @jfiala (2017/08) @lukoyanov (2017/09) @cbornet (2017/09) @jeff9finger (2018/01) @karismann (2019/03) @Zomzog (2019/04) @lwlee2608 (2019/10) @bkabrda (2020/01)

### CODE REVIEW

**Comment by wolfdale** [comment]
@wing328 Can you please take a look at this PR. Thanks!

**Comment by wolfdale** [comment]
@wing328 gentle remainder. Can we get this in master?

File: samples/server/petstore/jaxrs-spec-interface/src/gen/java/org/openapitools/model/ArrayTest.java [hunk 1]
```diff
@@ -59,7 +77,25 @@ public ArrayTest arrayArrayOfInteger(List<List<Long>> arrayArrayOfInteger) {
 
   public void setArrayArrayOfInteger(List<List<Long>> arrayArrayOfInteger) {
     this.arrayArrayOfInteger = arrayArrayOfInteger;
-  }/**
+  }
+
```
Comment (wing328) [inline samples/server/petstore/jaxrs-spec-interface/src/gen/java/org/openapitools/model/ArrayTest.java:81]: Minor suggestion: can you please also add proper docstrings for these new methods in another PR?

**Review by wing328 — APPROVED** [review]
LGTM but I've not tested it locally.

**Comment by rastislavpapp** [comment]
Is there an option to disable this? I want just plain POJOs, these crazy methods break a lot of things (mapstruct for example).

**Pull request merged**